central/graphql/resolvers/image_components.go

```diff
@@ -504,16 +496,9 @@ func (resolver *imageComponentResolver) LayerIndex() (*int32, error) {
 		return &v, nil
 	}
 
-	var scope scoped.Scope
-	var hasScope bool
-	if features.VulnMgmtWorkloadCVEs.Enabled() {
-		if scope, hasScope = scoped.GetScopeAtLevel(resolver.ctx, v1.SearchCategory_IMAGES); !hasScope {
-			return nil, nil
-		}
-	} else {
-		if scope, hasScope = scoped.GetScope(resolver.ctx); !hasScope || scope.Level != v1.SearchCategory_IMAGES {
-			return nil, nil
-		}
+	scope, hasScope := scoped.GetScopeAtLevel(resolver.ctx, v1.SearchCategory_IMAGES)
+	if !hasScope {
+		return nil, nil
 	}
```

Comment:
This is the only BE change that is more than the most trivial `if` block deletion.

Reply:
LGTM